md/work-execution-audit.md

```diff
@@ -0,0 +1,61 @@
+# Work execution audit
+
+Sometimes it is useful to understand what is currently happening in the platform, in particular for work execution, see [Work service mechanism](execution-sequence-states-and-transactions.md).
+
+This can be achieved using a specific logger called _Work execution audit_
+
+
+## How to activate it
+
+### Enable the logger
+
+Enable the logger called `BONITA_WORK_AUDIT.EXECUTION` in the logger configuration.
```

Comment:
This should contain actual examples of logging configuration for Tomcat (in logging.properties) and Wildfly (in standalone.xml).